packages/now-cli/src/commands/env/add.ts

```diff
@@ -96,7 +96,9 @@ export default async function add(
 
   let envValue: string;
 
-  if (stdInput) {
+  if (isSystemEnvVariable(envName)) {
+    envValue = '';
+  } else if (stdInput) {
     envValue = stdInput;
   } else {
```

Comment:
I would suggest switch the order of these two conditions, so that we fail if you add a system env variable with `stdInput` (the api call will fail with `${key} is a System Environment Variable and cannot have a value.`).

Reply:
I had it that way at first but then switched it 😆